holopy/scattering/calculations.py

```diff
@@ -51,7 +56,7 @@ def interpret_theory(scatterer,theory='auto'):
 
 def finalize(schema, result):
     if not hasattr(schema, 'flat'):
-        result = from_flat(result)
+        result=from_flat(result)
```

Comment:
Convention is to put spaces around binary operators (like =) except in when assigning to function arguments inside the parens, so probably put these spaces back unless you had a good reason for removing them.